user.py

```diff
@@ -1,15 +1,31 @@
 from decorators import login_required
-from db import db, User
-from flask import Blueprint, render_template, flash, current_app, url_for, request, make_response, redirect, session
-import re
+from db import db, User, Researcher, Course, PreferenceAssignment
+from data import RESEARCHERS_TYPE
+from flask import Blueprint, render_template, flash, url_for, request, make_response, redirect, session, \
+    Flask
+import re, json
 
 user_bp = Blueprint('user', __name__)
 
+app = Flask(__name__)
+app.config.from_file("config.json", load=json.load)
+
+
+def get_default_max_load(researcher_type):
```

Comment:
Maybe an enum would be more resilient as runtime as it won't even load if the enum does not exist.